src/bench/checkblock.cpp

```diff
@@ -8,11 +8,36 @@
 #include <validation.h>
 #include <streams.h>
 #include <consensus/validation.h>
+#include <rpc/blockchain.h>
+
+#include <univalue.h>
 
 namespace block_bench {
 #include <bench/data/block413567.raw.h>
 } // namespace block_bench
 
+static void SerializeJSONBlockTest(benchmark::State& state) {
```

Comment:
This could be in a new benchmark, for instance, `src/bench/rpc_blockchain.cpp`.

Reply:
nit, could rename to `BlockToJSONVerbose`.

Reply:
@promag moved and renamed